src/test/regress/expected/multi_mx_create_table.out

```diff
@@ -484,3 +484,65 @@ ORDER BY colocationid, logicalrelid;
  articles_single_shard_hash_mx                          |      1390011 |           1 | h          | s
 (23 rows)
 
+-- check the citus_tables view
+SELECT table_name, citus_table_type, distribution_column, shard_count, table_owner, access_method
+FROM citus_tables
+ORDER BY table_name::text;
+                       table_name                       | citus_table_type | distribution_column | shard_count | table_owner | access_method
+---------------------------------------------------------------------
+ app_analytics_events_mx                                | distributed      | app_id              |           4 | postgres    | heap
+ articles_hash_mx                                       | distributed      | author_id           |           2 | postgres    | heap
+ articles_single_shard_hash_mx                          | distributed      | author_id           |           1 | postgres    | heap
+ citus_mx_test_schema.nation_hash                       | distributed      | n_nationkey         |          16 | postgres    | heap
+ citus_mx_test_schema.nation_hash_collation_search_path | distributed      | n_nationkey         |           4 | postgres    | heap
+ citus_mx_test_schema.nation_hash_composite_types       | distributed      | n_nationkey         |           4 | postgres    | heap
+ citus_mx_test_schema_join_1.nation_hash                | distributed      | n_nationkey         |           4 | postgres    | heap
+ citus_mx_test_schema_join_1.nation_hash_2              | distributed      | n_nationkey         |           4 | postgres    | heap
+ citus_mx_test_schema_join_2.nation_hash                | distributed      | n_nationkey         |           4 | postgres    | heap
+ company_employees_mx                                   | distributed      | company_id          |           4 | postgres    | heap
+ customer_mx                                            | reference        |                     |           1 | postgres    | heap
```

Comment:
Minor: Maybe we can have a placeholder (for distribution column) here instead of being empty? "N/A" or something like that

One thing that might be confusing for users is if we have a citus_local table then its type will also be shown distributed but it won't have a distribution column

Reply:
will change it to `<none>`